tests/ont/test_generate_email.py

```diff
@@ -58,16 +59,15 @@ def test_generate_email(self):
             == f"Update: ONT run {expt} flowcell {flowcell_id} has been {event_type}"
         )
 
-        study_lines = "\n".join(studies)
+        study_descs = [f"{s.id_study_lims} ({s.name})" for s in studies]
+        study_lines = "\n".join(study_descs)
 
-        assert event.body(*studies) == (
-            f"The ONT run for experiment {expt}, flowcell {flowcell_id} has been {event_type}.\n"
-            "The data are available in iRODS at the following path:\n"
+        assert event.body(studies) == (
+            f"The ONT run for experiment {expt}, flowcell {flowcell_id} has been {event_type}. The data are available in iRODS at the following path:\n"
             "\n"
             f"{path}\n"
             "\n"
-            "This is an automated email from NPG. You are receiving it because you are registered\n"
-            "as a contact for one or more of the Studies listed below:\n"
+            "This is an automated email from NPG. You are receiving it because you are registered as a contact for one or more of the Studies listed below:\n"
```

Comment:
Notifications for Illumina and PacBio platforms have a standard footer. Do we really want ONT notifications to be different?

Reply:
Yes. Lines 152-154 were added on request by SSRs

Reply:
I'm also expecting a few other details in the next week or two after it's live - listing the plate barcodes and tag names/sequences.

Reply:
That's fine.